README.md

```diff
@@ -1,2 +1,15 @@
-# osrm-openapi
-OpenAPI spec for generating OSRM Models
+# OSRM OpenAPI
+
+OpenAPI spec and generation script to build OSRM Models for various platforms.
+
+## Building OSRM Models
```

Comment:
Probably add a few words about dependencies. I'm assuming just `brew install` the latest openapi-generator or something.